state/core_access.go

```diff
@@ -459,6 +524,53 @@ func (ca *CoreAccessor) QueryRedelegations(
 	})
 }
 
+func (ca *CoreAccessor) LastPayForBlob() int64 {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	return ca.lastPayForBlob
+}
+
+func (ca *CoreAccessor) PayForBlobCount() int64 {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	return ca.payForBlobCount
+}
+
+func (ca *CoreAccessor) markSuccessfulPFB() {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	ca.lastPayForBlob = time.Now().UnixMilli()
+	ca.payForBlobCount++
+}
+
+func (ca *CoreAccessor) setMinGasPrice(minGasPrice float64) {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	ca.minGasPrice = minGasPrice
+}
+
+func (ca *CoreAccessor) getMinGasPrice() float64 {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	return ca.minGasPrice
+}
```

Comment:
Does not it deserve to be a separate data structure inside the coreAccesor? Imo would be much cleaner

Reply:
We can park this in the issue

Reply:
ok